azurerm/internal/services/web/tests/resource_arm_app_service_hybrid_connection_test.go

```diff
@@ -0,0 +1,145 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMAppServiceHybridConnection_basic(t *testing.T) {
```

Comment:
we need to add support for & a requires import test here - see other resources for an example (this forms part of #2807)